bookwyrm/activitypub/base_activity.py

```diff
@@ -73,7 +73,7 @@ class ActivityObject:
     def __init__(
         self,
         activity_objects: Optional[list[str, base_model.BookWyrmModel]] = None,
-        **kwargs: dict[str, Any],
+        **kwargs: Any,
```

Comment:
This change seems good— why not use it for all of _args/kwargs_, below?

Reply:
This file itself is not checked with mypy yet, so I just added type annotations to functions that are called from files that are already checked.

Reply:
Thanks.